component/common/src/main/java/org/wso2/carbon/extension/identity/emailotp/common/util/Utils.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.extension.identity.emailotp.common.util;
+
+import org.apache.commons.codec.digest.DigestUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.extension.identity.emailotp.common.constant.Constants;
+import org.wso2.carbon.extension.identity.emailotp.common.dto.ConfigsDTO;
+import org.wso2.carbon.extension.identity.emailotp.common.exception.EmailOtpClientException;
+import org.wso2.carbon.extension.identity.emailotp.common.exception.EmailOtpServerException;
+import org.wso2.carbon.extension.identity.emailotp.common.internal.EmailOtpServiceDataHolder;
+import org.wso2.carbon.identity.event.IdentityEventConfigBuilder;
+import org.wso2.carbon.identity.event.IdentityEventException;
+import org.wso2.carbon.identity.event.bean.ModuleConfiguration;
+
+import java.util.Properties;
+import java.util.UUID;
+
+/**
+ * Util functions for Email OTP service.
+ */
+public class Utils {
+
+    private static final Log log = LogFactory.getLog(Utils.class);
+
+    /**
+     * Read configurations and populate ConfigsDTO object.
+     *
+     * @throws EmailOtpServerException Throws upon an issue while reading configs.
+     */
+    public static void readConfigurations() throws EmailOtpServerException {
+
+        Properties properties;
+        try {
+            ModuleConfiguration configs = IdentityEventConfigBuilder.getInstance()
+                    .getModuleConfigurations(Constants.EMAIL_OTP_IDENTITY_EVENT_MODULE_NAME);
+            if (configs != null) {
+                properties = configs.getModuleProperties();
+            } else {
+                properties = new Properties();
+                log.debug("Couldn't find Email OTP handler configurations.");
```

Comment:
Is this s debug log or an error log?

Reply:
Addressed in 148a8f957bdd2c4d574fb53e81284dfc017db263 , This is debug log and checked `isDebugEnabled`.